src/app/search/query-builder.service.ts

```diff
@@ -65,26 +65,24 @@ export class QueryBuilderService {
     return body.rawOption('_source', false);
   }
 
-  private excludeVerifiedContent(body: any) {
-    // TODO: it should be possible to exclude tags and workflowVersions too
-    // however, it currently breaks the contents of the datatables since verified information is not aggregated by
-    // tool or workflow properly.
-    return body.rawOption('_source', {
-      excludes: [
-        '*.content',
-        '*.sourceFiles',
-        'description',
-        'users',
-        'workflowVersions.dirtyBit',
-        'workflowVersions.hidden',
-        'workflowVersions.last_modified',
-        'workflowVersions.name',
-        'workflowVersions.valid',
-        'workflowVersions.workflow_path',
-        'workflowVersions.workingDirectory',
-        'workflowVersions.reference',
-      ],
-    });
+  // These are the properties to return in the search to display the results table correctly
+  private sourceOptions(body: any) {
+    return body.rawOption('_source', [
+      'author',
```

Comment:
I wish these strings weren't hard-coded like this, but I guess there's no easy solution. Also, was like this already.